### PR TITLE
Make npm run clean platform agnostic

### PR DESCRIPTION
**Pull request by Stanzilla**
meh-ish solution since it installs rimraf globally, open to other suggestions

`npm cache clear` was removed because it is no longer needed with npm@5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,10 +10,10 @@
     "test:unit": "ava test/unit",
     "test:unit:watch": "npm run test:unit -- --watch",
     "prepush": "npm test",
-    "postinstall": "install-app-deps && npm run rebuild-node-pty",
+    "postinstall": "install-app-deps && npm run rebuild-node-pty && npm install -g rimraf",
```
Comment (chabou) [inline package.json]: you don't need to install it globally. Add it as a dev dependency

Comment (Stanzilla) [inline package.json]: @chabou that won't work since it will try to delete itself in that step

**Comment by Stanzilla** [comment]
The other idea would be to just make a `wclean` task that uses Windows tools.

File: package.json (hunk 1 above)
Comment (albinekb) [inline package.json]: we can't merge this if it installs it globally 🌍

**Comment by chabou** [comment]
Or maybe use: https://www.npmjs.com/package/rimraf-standalone

**Comment by Stanzilla** [comment]
That would mean having rimraf-standlone.js in the repo, would you guys be fine with that?

File: package.json (hunk 1 above)
Comment (Stanzilla) [inline package.json]: I know, that's why I have been asking for other options :)

**Comment by chabou** [comment]
I don't know if I'm fine with that but it should be in a `./bin/` directory in any case 😛

**Comment by Stanzilla** [comment]
Thoughts?

**Pull request merged**